jina/drivers/predict.py

```diff
@@ -10,9 +10,15 @@
 
 
 class BasePredictDriver(FastRecursiveMixin, BaseExecutableDriver):
-    """Drivers inherited from this Driver will bind :meth:`predict` by default """
+    """Drivers inherited from this Driver will bind :meth:`predict` by default
```

Comment:
Instead of `this` driver use the class name `BasePredictDriver`. Nice work on this